src/main/java/at/hannibal2/skyhanni/data/GardenCropMilestones.kt

```diff
@@ -51,6 +61,73 @@ object GardenCropMilestones {
         GardenCropMilestonesCommunityFix.openInventory(event.inventoryItems)
     }
 
+    /**
+     * Just for me testing
+     */
+    var enabled = false
+    var menabled = false
+    var curr = 0L
+    var mcurr = 0L
+    var add = 0L
+    var madd = 0L
+
+    @SubscribeEvent
+    fun onTick(event: LorenzTickEvent) {
+        if (enabled) {
+            val stack = InventoryUtils.getItemInHand()
+            val crop = stack?.getCropType() ?: return
+            val old = crop.getCounter()
+            val toAdd = old + add
+            crop.setCounter(toAdd)
+            curr = toAdd
+        } else if (menabled) {
+            val mush = CropType.MUSHROOM
+            val mold = mush.getCounter()
+            val mtoAdd = mold + madd
+            mush.setCounter(mtoAdd)
+            mcurr = mtoAdd
+        }
+    }
+
+    @SubscribeEvent
+    fun onCropMilestoneOverflowLevelUpEvent(event: CropMilestoneOverflowLevelUpEvent) {
+        val crop = event.cropType
+        val oldLevel = event.oldLevel
+        val newLevel = event.newLevel
+        val customGoalLevel = ProfileStorageData.profileSpecific?.garden?.customGoalMilestone?.get(crop) ?: 0
+        val goalReached = newLevel == customGoalLevel
+
+        val rewards = buildList {
```

Comment:
all this code below looks awfully similar to the skill overflow message. write a u common utils class for overflow level up messages?